atlasdb-jdbc/src/main/java/com/palantir/atlasdb/keyvalue/jdbc/JdbcKeyValueService.java

```diff
@@ -641,6 +643,22 @@ public Void apply(DSLContext ctx) {
     }
 
     @Override
+    public void deleteRange(TableReference tableRef, RangeRequest range) {
```

Comment:
Just kidding, it's here because JDBCKVS is dumb and doesn't extend AbstractKVS, similarly to RocksDBKVS